src/styles/_variables.scss

```diff
@@ -35,6 +35,7 @@ $font-condensed: "PT Sans Narrow", $font-default;
 $text-default-font-size: 16px;
 $text-sm-font-size: 14px;
 $text-xs-font-size: 12px;
+$product-item-default-name-font-size: 18px;
```

Comment:
I think there will be something general like 
`$text-m-font-size: 18px;` above. That you will use in all app. Always think about the whole app, now and in future.